website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,231 @@
+**Order**: Verdaccio@4.x has basic support for package ordering from `config.yaml`. The package list can be sorted ascending & descending. [Find out more](https://verdaccio.org/docs/en/webui#configuration)
+
+### Detailed Page
+The new Detailed package in a more categorized manner for readme, dependencies, version and uplinks. 
+
+![alt text](./assets/detail-page.png "verdaccio@4 detail page")
+
+### Package Sidebar
+
+The Package Sidebar includes most relevant information from package metadata. You can open an issue, see Readme and download the package tarball. It also clearly shows the package's minimum requirements on node and npm. 
+
+Also, The package sidebar shows *Author*, *Maintainers* and *Contributors* in different sections. When you click on person avatar, you'll be able to contact that person via email.
+
+## New Browser Router
+Till, verdaccio@3.x we have Hash Router implementation on frontend application routes. We faced a lot of problem with hash router in the Readme section. The Readme also uses (#) hash for the heading tags and anchor elements. 
+
+In Verdaccio@4.x, we migrated the Hash Router to Browser Router with a more cleaner look. (No more hashes in URLs).
+
+## Unpublish Role
+
+Verdaccio@4.x improves the package management by adding an access layer to publish and unpublish. Now you can have restrictions to some of the users for publishing and unpublishing. [Find out more](https://verdaccio.org/docs/en/packages )
+
+## Disable Gravatar
+Verdaccio uses [Gravatar](https://en.gravatar.com) to show the images of authors, contributors and maintainers. Now, gravatar support can be disabled from Verdaccio `config.yaml`. 
+
+```yaml
+web:
+  title: Verdaccio
+  gravatar: false
+```
+
+In order to be fully offline, The fallback support is a generic user face SVG based on base64.
+
+## New CLI Commands
+
+We are really excited to add some npm cli commands to Verdaccio. Now you can use `npm star`, and `npm profile`.
+
+### npm star
+
+Now a user can mark their favorite package. 
+
+```
+npm star [<package>..]
+```
+### npm profile
+
+With npm profile, a user can change their profile settings. 
+
+*Note:* Verdaccio does not support two-factor authentication yet.
+
+```
+npm profile get [--json|--parseable] [<property>]
+npm profile set [--json|--parseable] <property> <value>
+npm profile set password
+```
+
+Check out more at [https://docs.npmjs.com/cli/profile](https://docs.npmjs.com/cli/profile)
+
+
+## JWT Token
+
+Verdaccio supports [JSON Web Tokens](https://jwt.io/) for the authentication. The previous version of Verdaccio used `AES` token generator. The new JWT token standardizes the process and provides an additional mechanism for token generation. Verdaccio@4.x still supports the `AES` token generator.
+
+[Click here for more information on new JWT tokens](https://medium.com/verdaccio/diving-into-jwt-support-for-verdaccio-4-88df2cf23ddc)
+
+## Docker Improvements
+- https://github.com/verdaccio/verdaccio/pull/845
+@juanpicado
```

Comment:
```suggestion
There is no doubt that Docker has been a major breakthrough for this project, it's by far the most popular way to download Verdaccio, we have more than [4.200.000 downloads at this writing](https://dockeri.co/image/verdaccio/verdaccio) and for such reason, we care about improving the developer experience adding new features.

 [Please click here more information on the new Docker Image.](https://verdaccio.org/blog/2019/05/13/the-new-docker-image-verdaccio-4)
```

Reply:
I was thinking about an intro text to the other article, and you read my mind 🤣

Reply:
![muñeco feliz](https://user-images.githubusercontent.com/558752/58369945-be8b3600-7f01-11e9-8a13-2ce4a6538f75.gif)